app/views/admin/_menu.html.erb

```diff
@@ -158,8 +158,8 @@
         <span class="icon-settings"></span>
         <strong><%= t("admin.menu.title_site_customization") %></strong>
       </a>
-      <ul <%= "class=is-active" if menu_profiles? %>>
-        <li <%= "class=active" if controller_name == "pages" %>>
+        <ul <%= "class=is-active" if menu_customization? %>>
```

Comment:
@iagirre have you configured your Netbeans with editorconfig plugin? https://github.com/welovecoding/editorconfig-netbeans#readme that <ul> and <li> indentation seems off compared to the rest of the html

Reply:
Yes, I configured it, but there must be something wrong with it. I think it's OK now.